configs/raspberrypi_config

```diff
@@ -4,6 +4,9 @@
 # Raspberry Pi does not support GRUB bootloader integration, fallback to U-boot.
 MENDER_GRUB_EFI_INTEGRATION=n
 
+# Use same boot point that upstream to support Headless configuration
+MENDER_BOOT_PART_MOUNT_LOCATION="/boot"
```

Comment:
Hmm, will this work? The reason that we don't mount the boot partition directly on `/boot` is that the kernel is hosted in `/boot`, but it is still just a directory in the rootfs. If we mount the boot partition here, then you cannot update your kernel with a rootfs update.

Reply:
It boots and works, but I think you are right with regards to an update with contents in /boot (i.e. Kernel update). TBH I implemented what we said in the JIRA task, but I'll rethink a bit more now.